docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -285,4 +294,118 @@ Options are available to customize how targeting evaluation is performed across
     public TargetingFilter targetingFilter(ITargetingContextAccessor contextAccessor) {
         return new TargetingFilter(contextAccessor, new TargetingEvaluationOptions().setIgnoreCase(true));
     }
-----
\ No newline at end of file
+----
+
+=== Evaluating Dynamic Features
+
+Dynamic Features are enabled through the creation <<Feature Variants, feature variants>>, where each variant describes who is assigned that variant, and a reference to the configurations that make that variant. DynamicFeatures are accessed through the `DynamicFeatureManager`.
```

Comment:
I would reiterate here that a variant is one of multiple options for a feature to make sure people understand the full picture.